### DISCUSSION TITLE
Oqtane & DMZ or DR Servers

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#2264] [General]
Hi,

Oqtane can run DMZ or DR servers? Any tweaks has to make in the code level?

Thanks & Regards!

**Comment by sbwalker** [comment]
That is like asking if a car can be driven off road - its impossible to answer without knowing the detailed specifics of the environment.

**Reply by Sharafudeen** [reply]
Haha..
Actually we have internal organization users and external users (who hired temporary). We will have a Oqtane based application which would support both types of users in a single site / application. For external users, we might host (part of the application / modules) in the DMZ servers. We know DMZ servers requires proxy to access the application from external. Is it possible to do it in Oqtane framework without doing much code in the framework level?
